src/tabnineChatWidget/ChatApi.ts

```diff
@@ -148,6 +152,10 @@ export function initChatApi(
       "peek_definition",
       peekDefinition
     )
+    .registerEvent<NavigateToLocationPayload, void>(
+      "navigate_to_location",
+      navigateToLocation
```

Comment:
and if you do ned to avoid on it, it would be better to do it here

Reply:
what?